package.json

```diff
@@ -32,7 +32,7 @@
     "react-router": "^4.2.0",
     "react-test-renderer": "^15.6.2",
     "shelljs": "^0.7.8",
-    "truffle": "^3.4.9",
+    "truffle": "^3.4.11",
```

Comment:
Do we need the old version of Truffle here as we install the last version of it before each test?